packages/edit-post/src/components/sidebar/post-title/style.scss

```diff
@@ -0,0 +1,3 @@
+.edit-post-post-title {
+	font-size: 2em;
```

Comment:
@jasmussen this is a `span` for the post title, that will need the desired styles 😄

Reply:
Excellent let me take a stab in a minute, thank you!